File: C++ coding with Muhammad Javed/21 to 30 tut/main.cpp
#include <iostream>
#include <cmath>
#include <string>

using namespace std;

int main()
{
    int x1, x2, y1, y2;

    cout << "Enter the value of x1: ";
    cin >> x1;
    cout << "Enter the value of y1: ";
    cin >> y1;
    cout << "Enter the value of x2: ";
    cin >> x2;
    cout << "Enter the value of :y2 ";
    cin >> y2;

    cout << "The point 1 is (" << x1 << ", " << y1 << ")" << endl;
    cout << "The point 2 is (" << x2 << ", " << y2 << ")" << endl;

    double dist = sqrt(pow(x1 - x2, 2) + pow(y1 - y2, 2));

    cout << "The distance between point" << "(" << x1 << ", " << y1 << ") and (" << x2 << ", " << y2 << ") is " << dist << endl;

    return 0;
}

File: C++ coding with Muhammad Javed/21 to 30 tut/main.py
x1 = int(input("Enter value of x1 : "))
x2 = int(input("Enter value of x2 : "))
y1 = int(input("Enter value of y1 : "))
y2 = int(input("Enter value of y2 : "))


print(f"The point is 1 is ({x1} , {y1})")
print(f"The point is 2 is ({x2} , {y2})")

dist  = ((((x1-x2)**2) + ((y1-y2)**2) )**0.5)


print(f"The distance between point ({x1} , {y1}) and ({x2} , {y2}) is {dist}")

